src/shared/components/Icon.tsx

```diff
@@ -0,0 +1,13 @@
+type IconProps = {
+	name: string;
+	size?: number;
+	className?: string;
+};
+
+export default function Icon({ name, size = 24, className }: IconProps) {
+	return (
+		<svg width={size} height={size} className={className} aria-hidden="true">
+			<use href={`#icon-${name}`} />
+		</svg>
+	);
+}
```

Comment:
```suggestion
const Icon = ({ name, size = 24, className }: IconProps) => {
  return (
    <svg width={size} height={size} className={className} aria-hidden="true">
      <use href={`#icon-${name}`} />
    </svg>
  );
};

export default Icon;
```

arrow function 사용하기 !